### PR TITLE
haskell-stack 0.1.8.0

### PR DESCRIPTION
**Pull request by borsboom**
Since Stack no longer uses the `ignore` package, the dependence on libpcre is removed.

### CODE REVIEW

File: Library/Formula/haskell-stack.rb [hunk 1]
```diff
@@ -4,9 +4,9 @@ class HaskellStack < Formula
   include Language::Haskell::Cabal
 
   desc "The Haskell Tool Stack"
-  homepage "https://www.stackage.org"
-  url "https://hackage.haskell.org/package/stack-0.1.6.0/stack-0.1.6.0.tar.gz"
-  sha256 "a47ffc204b9caef8281d1e5daebc21bc9d4d2414ed695dc10d32fcca4d81978d"
+  homepage "https://github.com/commercialhaskell/stack"
```
Comment (DomT4) [inline Library/Formula/haskell-stack.rb]: Why the change here? Upstream's Github repo still seem to point to the existing one as the homepage?

Comment (borsboom) [inline Library/Formula/haskell-stack.rb]: That was a historical detail (an earlier incarnation of stack was called
stackage-build before the two projects were separated), and as of now no
longer does.  The `stack.cabal` file has pointed to
https://github.com/commercialhaskell/stack for a long time, as do the
`.deb`s and `.rpms`.

>

Comment (DomT4) [inline Library/Formula/haskell-stack.rb]: Cool. Thanks for the clarification!

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/5f8969f3696a076946ac68191f31bb1f640e4ce3. Thank you for another Homebrew contribution @borsboom! :smiley_cat:
